osu.Game/Overlays/Settings/SettingsItem.cs

```diff
@@ -100,10 +100,9 @@ public virtual Bindable<T> Current
 
         public IEnumerable<string> Keywords { get; set; }
 
-        public bool MatchingFilter
-        {
-            set => Alpha = value ? 1 : 0;
-        }
+        public override bool IsPresent => MatchingFilter;
```

Comment:
Is bypassing the base property entirely intentional here? I'd expect this to be written as

```suggestion
        public override bool IsPresent => base.IsPresent && MatchingFilter;
```

because otherwise you could e.g. fade a `SettingsItem` out but it would still continue updating in the background.

Same thing goes for this override in `SettingsSection`.

Reply:
It was intentional to avoid weird execution order issues. I can try bringing back base if it's seen as bad (may work after my other more recent surrounding changes).